src/main/java/core/basesyntax/FigureDrawer.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureDrawer {

```

Comment:
Bad interface name - this interface doesn't know anything about figures.
I would use Drawable name in this case